mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -115,16 +139,31 @@ private object ScalaConverter {
     totalDeletedSize = event.getTotalDeletedSize,
     mailboxId = event.getMailboxId)
 
+  private def toScala(event: JavaAdded): DTO.Added = DTO.Added(
+    sessionId = event.getSessionId,
+    user = event.getUser,
+    path = DTO.MailboxPath.fromJava(event.getMailboxPath),
+    mailboxId = event.getMailboxId,
+    added = event.getAdded.asScala.map(entry => entry._1 -> MetaDataDTO.MessageMetaData.fromJava(entry._2)).toMap)
```

Comment:
what about mapValues ?

Reply:
Not really in my changeset, I just moved the `)` closing bracket.

I will let Duc handle this here https://github.com/linagora/james-project/pull/2039#discussion_r242079318 and here https://github.com/linagora/james-project/pull/2039/commits/f33dcd2183a898482a2079034c8131a1d7354f9b